sda-commons-web-autoconfigure/src/main/java/org/sdase/commons/spring/boot/web/auth/SdaSecurityConfiguration.java

```diff
@@ -83,17 +84,29 @@ AuthenticationManager authenticationManager(AuthenticationManagerBuilder auth) {
 
   private void oidcAuthentication(HttpSecurity http) throws Exception {
     var authenticationManagerResolver = createAuthenticationManagerResolver();
-    http.csrf()
-        .disable() // NOSONAR
-        .authorizeRequests(
-            authorize ->
-                authorize.anyRequest().permitAll().accessDecisionManager(sdaAccessDecisionManager))
+
+    http.authorizeHttpRequests(
+            authorize -> authorize.requestMatchers("/**").access(sdaAccessDecisionManager))
         .oauth2ResourceServer(
             oauth2 -> oauth2.authenticationManagerResolver(authenticationManagerResolver))
         .headers(
             configurer ->
                 configurer.addHeaderWriter(
                     new StaticHeadersWriter(sdaSecurityHeaders.getSecurityHeaders())));
+
+    /*
+    http.csrf()
+       .disable() // NOSONAR
+       .authorizeRequests(
+           authorize ->
+               authorize.anyRequest().permitAll().accessDecisionManager(sdaAccessDecisionManager))
+       .oauth2ResourceServer(
+           oauth2 -> oauth2.authenticationManagerResolver(authenticationManagerResolver))
+       .headers(
+           configurer ->
+               configurer.addHeaderWriter(
+                   new StaticHeadersWriter(sdaSecurityHeaders.getSecurityHeaders())));
+    */
```

Comment:
Is there a reason to remove the csrf config?
Imho CRSF protection (with a token as in this implementation) does not make sense for stateless REST APIs

```suggestion
                    new StaticHeadersWriter(sdaSecurityHeaders.getSecurityHeaders())))
        .csrf()
        .disable();
```